docs/content/get-started/integrations/flow-control/sdk/java/auto-instrumentation.md

```diff
@@ -80,4 +80,4 @@ aperture.javaagent.ignored.paths=/health,/connected
 ```
 
 [download_link]:
-  https://repo1.maven.org/maven2/com/fluxninja/aperture/aperture-javaagent/1.0.0/aperture-javaagent-1.0.0.jar
+  https://repo1.maven.org/maven2/com/fluxninja/aperture/aperture-javaagent/1.0.0
```

Comment:
This could be set to 1.3.0 instead of 1.0.0 right now; I'll update the link to 1.5.0 when I release it

Reply:
can we somehow point to the latest tag all the time?

Reply:
let me just remove the version number

Reply:
I don't think it's possible to provide a 'latest' maven jar link. We can point to maven page with all versions (https://search.maven.org/artifact/com.fluxninja.aperture/aperture-javaagent), but I aimed for it to be a simple link to just download the jar.

We could publish it somewhere outside of maven to keep a 'latest' link

Reply:
somewhere outside maven, e.g. github releases page